### PR TITLE
Change name of downloaded file when exporting study

### PR DESCRIPTION
**Pull request by souissimai**

### CODE REVIEW

File: src/components/dialogs/export-dialog.jsx [hunk 1]
```diff
@@ -104,16 +132,21 @@ const ExportDialog = ({
...
                 urlSearchParams.append('formatParameters', jsoned);
-                // we have already as parameters, the tokens, so use '&' in stead of '?'
-                suffix = '&' + urlSearchParams.toString();
+            }
+            if (!isBlankOrEmpty(studyName)) {
+                urlSearchParams.append('studyName', studyName);
             }
 
+            // we have already as parameters, the access tokens, so use '&' instead of '?'
+            suffix = urlSearchParams.toString()
```
Comment (SlimaneAmar) [inline src/components/dialogs/export-dialog.jsx:145]: Need to test with an empty urlSearchParams

File: src/components/dialogs/export-dialog.jsx [hunk 2]
```diff
@@ -54,9 +60,31 @@ const ExportDialog = ({
...
     const [unfolded, setUnfolded] = React.useState(false);
 
+    const fetchStudyName = useCallback(() => {
+        fetchDirectoryElementPath(studyUuid)
+            .then((response) => {
+                const studyName = response[response.length - 1]?.elementName;
+                setStudyName(studyName);
+            })
+            .catch((error) => {
+                snackError({
```
Comment (SlimaneAmar) [inline src/components/dialogs/export-dialog.jsx:75]: Need to test the error

File: src/components/dialogs/export-dialog.jsx (hunk 1 above)
Comment (souissimai) [reply src/components/dialogs/export-dialog.jsx:145]: tested

File: src/components/dialogs/export-dialog.jsx (hunk 2 above)
Comment (souissimai) [reply src/components/dialogs/export-dialog.jsx:75]: tested 
fail

**Review by SlimaneAmar — APPROVED** [review]
